src/test/librbd/object_map/test_mock_UpdateRequest.cc

```diff
@@ -232,14 +232,14 @@ TEST_F(TestMockObjectMapUpdateRequest, BatchUpdate) {
                                         no_progress));
   ASSERT_EQ(0, acquire_exclusive_lock(*ictx));
 
+  expect_unlock_exclusive_lock(*ictx);
   InSequence seq;
   expect_update(ictx, CEPH_NOSNAP, 0, 262144, OBJECT_NONEXISTENT, OBJECT_EXISTS,
                 0);
   expect_update(ictx, CEPH_NOSNAP, 262144, 524288, OBJECT_NONEXISTENT,
                 OBJECT_EXISTS, 0);
   expect_update(ictx, CEPH_NOSNAP, 524288, 712312, OBJECT_NONEXISTENT,
                 OBJECT_EXISTS, 0);
-  expect_unlock_exclusive_lock(*ictx);
```

Comment:
Nit: why did this need to change for the object-map?

Reply:
I moved it before expect_update().